source/val/validate_conversion.cpp

```diff
@@ -469,7 +469,7 @@ spv_result_t ConversionPass(ValidationState_t& _, const Instruction* inst) {
                << "or scalar: " << spvOpcodeString(opcode);
 
       if (_.version() >= SPV_SPIRV_VERSION_WORD(1, 5) ||
-          _.HasExtension(kSPV_KHR_physical_storage_buffer)) {
+          _.HasExtension(kSPV_EXT_physical_storage_buffer)) {
```

Comment:
Is this needed?

Reply:
Yes, it was modified in the headers somehow. Couldn't pin the exact place it was changed